README.md

```diff
@@ -9,52 +9,48 @@
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Citation](https://img.shields.io/badge/cite-citation-blue)](https://arxiv.org/abs/2105.06413)
 
+## About
 
-[Federated learning](https://en.wikipedia.org/wiki/Federated_learning) is a distributed machine learning approach that
-enables organizations to collaborate on machine learning projects
-without sharing sensitive data, such as, patient records, financial data,
-or classified secrets 
-([Sheller MJ,  et al., 2020](https://www.nature.com/articles/s41598-020-69250-1);
-[Sheller MJ, et al., 2019](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6589345);
-[Yang Y, et al., 2019](https://arxiv.org/abs/1902.04885);
-[McMahan HB, et al., 2016](https://arxiv.org/abs/1602.05629)).
+Open Federated Learning (OpenFL) enables data scientists to set up a federated learning experiment:
 
+- [aggregator-based workflow](https://openfl.readthedocs.io/en/docs_correction/source/workflow/running_the_federation.agg_based.html)
```

Comment:
This README will be many user's first introduction to OpenFL. These workflows should be named for what they do instead of the internal components they use. I would recommend changing aggregator based workflow to `Experiment Workflow` and director based workflow to `Session Workflow`